README.md

```diff
@@ -2,19 +2,19 @@
 
 A Ruby library for harvesting metadata from [OAI-PMH](https://www.openarchives.org/OAI/openarchivesprotocol.html) repositories.
 
-**Current version:** 0.5.0  
+**Current version:** 0.6.0
```

Comment:
The trailing spaces here are significant: they add a line break rather than a paragraph.